simple-social-icons.php

```diff
@@ -550,22 +563,22 @@ function css() {
 		$css = str_replace( "\t", '', $css );
 		$css = str_replace( array( "\n", "\r" ), ' ', $css );
 
-		echo '<style type="text/css" media="screen">' . $css . '</style>';
+		echo '<style type="text/css" media="screen">' . wp_strip_all_tags( $css ) . '</style>'; // phpcs:ignore
```

Comment:
`phpcs` still shows an error with `wp_strip_all_tags()`

But this is basically the same thing the 'Additional CSS' Customizer panel does, though it [raises an error](https://github.com/WordPress/wordpress-develop/blob/671d2caa29fd1dd13aa64be610f3b503d303685b/src/wp-includes/customize/class-wp-customize-custom-css-setting.php#L166) if there is a tag.